### DISCUSSION TITLE
Add more options to chart

### DISCUSSION THREAD

**Discussion by edmaSB** [orchidsoftware/platform#2855] [Ideas]
Would it be possible to add other custom options for the Charts? 
I'm working for a client who needs to use customized tooltips. I'm using an external component but it would be nice to have more options

**Comment by angtheod** [comment]
+1, my company also needs feature to be able to customize the data presented in Tooltips of the chart. Currently, in the tooltip you can only see the Y-axis value of the chart for that point.

But.. for this to happen I suggest you start using a more feature-rich and most importantly **Active** Frontend Library for Charts

Of course, keep [frappe/charts](https://github.com/frappe/charts) for backwards compatibility, but this node package looks abandond.
[Last release](https://github.com/frappe/charts/releases/tag/v1.6.3) was in Apr 2022 and before that it was on Apr 2021.

Pull Requests from Contributors are long forgotten... last comment from package author was in 2020.
There are 3 PRs related to this Idea that have not received any attention for years, so I believe it is safe to say this frappe/charts is abandond.

https://github.com/frappe/charts/pull/379 > Mar 2022
https://github.com/frappe/charts/pull/369 > Dec 2021
https://github.com/frappe/charts/pull/305 > Aug 2020

**Reply by tabuna** [reply]
It’s unfortunate to hear that the project has been abandoned. I had previously spoken with a Frappe employee who maintained this library about fixes and support.

I think we could switch to something more modern and actively supported, especially since I plan to work on the next version in the coming months.

If you're interested in the replacement, it would be great if you could help.

**Reply by angtheod** [reply]
Great, I am interested yea so will try to help with it, as soon as I have some free time.
Since, I am mainly a backend dev though, it's probably best if a frontend person work on it
